rmgpy/rmg/main.py

```diff
@@ -681,6 +682,13 @@ def execute(self, **kwargs):
                 )
                 
                 plotSensitivity(self.outputDirectory, index, reactionSystem.sensitiveSpecies)
+
+        # generate Cantera files chem.cti & chem_annotated.cti in a designated `cantera` output folder
+        try:
+            self.generateCanteraFiles(os.path.join(self.outputDirectory, 'chemkin', 'chem.inp'))
+            self.generateCanteraFiles(os.path.join(self.outputDirectory, 'chemkin', 'chem_annotated.inp'))
+        except:
```

Comment:
It's generally not a good idea to catch all exceptions in a try, except block because you may catch errors that you weren't expecting or are actual bad errors that should be fixed.

I think in this case, catching EnvironmentError might be ok, which includes any issues with reading/writing to the filesystem. Any potential issues with RMG or Cantera would still be raised, which we would then need to fix.